### ISSUE TITLE
Coverage using phpdbg on PHP 7.0.7 fails

### ISSUE THREAD

**Issue by bradynpoulsen** [sebastianbergmann/php-code-coverage#446]
Original report: sebastianbergmann/phpunit#2212

I just noticed that my code coverage fails when I use `phpdbg -qrr vendor/bin/phpunit` with my coverage flags. I continue to get the following error:

```
[PHP Fatal Error: Cannot declare (class/interface/trait) ________, because the name is already in use in ______ on line ___]
PHPUnit 5.4.4 by Sebastian Bergmann and contributors.

Runtime:       PHPDBG 7.0.7
Configuration: /path/to/phpunit.xml
```

I tested the following environments:

| Test Environment | PHP 7.0.6 | PHP 7.0.7 |
|-------------------------|---------------|---------------|
| **phpdbg** | Works | Fatal Error |
| **xdebug** | Works | Works |

**Comment by sebastianbergmann** [comment]
@bwoebi @krakjoe: Do you guys have an idea what's going on here?

**Comment by bwoebi** [comment]
Yes, it's a bug with went into the 7.0.7 release, which I only had fixed in master until getting a bug report… will be fixed in 7.0.8 … :-( see also https:bugs.php.net/72284

**Comment by sebastianbergmann** [comment]
Thank you, @bwoebi. Closing this issue as I cannot do anything about it.
